### PR TITLE
allow additional options to be passed into OpenSeadragon

### PR DESCRIPTION
**Pull request by locomo**
i took a stab at this: #1

### CODE REVIEW

File: src/openseadragon-curtain-sync.js [hunk 1]
```diff
@@ -436,7 +436,8 @@
         container: this.container,
         images: this.images,
         zoom: this.zoom,
-        pan: this.pan
+        pan: this.pan,
+        osdOptions: this.osdOptions
```
Comment (iangilman) [inline src/openseadragon-curtain-sync.js]: By passing this object in without copying it, we're allowing the modes to modify our original, which means the SyncMode will add `tileSources` that'll get picked up by the CurtainMode. The fix is just to copy it, which you can do with `extend`:

```
osdOptions: OpenSeadragon.extend({}, this.osdOptions)
```

File: src/openseadragon-curtain-sync.js [hunk 2]
```diff
@@ -27,10 +27,9 @@
 
     OpenSeadragon.EventSource.call(this);
 
-    this.viewer = OpenSeadragon({
-      element: args.container,
-      showNavigationControl: false
-    });
+    var ops = args.osdOptions;
+    ops.element = args.container;
+    this.viewer = OpenSeadragon( ops );
```
Comment (iangilman) [inline src/openseadragon-curtain-sync.js]: Not a big deal, but the code style in this library is for no spaces inside the parens (unlike the historic style of the OSD codebase).

**Comment by locomo** [comment]
ok cool .. i've made a couple of new commits updating the readme file and making the code changes you suggested!

**Comment by iangilman** [comment]
Looks fantastic! I think this is ready to merge, though of course it's up to you, @rpovelones.

@locomo thank you for doing this!

**Comment by rpovelones** [comment]
Hey guys, this all looks good! Thanks @locomo for all of your work here. I think it is a great addition! And thanks so much @iangilman for all of your advice and feedback! It is much appreciated! This is now merged!

**Comment by rpovelones** [comment]
Closes #1

**Comment by rpovelones** [comment]
Close #1

**Pull request merged**